CHANGELOG.md

```diff
@@ -2,35 +2,41 @@
 
 ## Next (Unreleased)
 
-## v.0.3.1 (October 1st, 2024)
+## v0.3.2 (October 15th, 2024)
+
+* Fallback to next auth mechanism if auth environment variables are set but empty ([#168](https://github.com/Kaggle/kagglehub/pull/168))
+* `kaggle.whoami()` returns proper value in Kaggle Notebook environment ([#166](https://github.com/Kaggle/kagglehub/pull/166))
+* `kaggle.whoami()` returns proper value when using Colab secrets ([#165](https://github.com/Kaggle/kagglehub/pull/165))
+
+## v0.3.1 (October 1st, 2024)
 
 * Attach competition in Kaggle notebook environment ([#161](https://github.com/Kaggle/kagglehub/pull/161))
 * Download competition using `kagglehub.competition_download(...)` ([#158](https://github.com/Kaggle/kagglehub/pull/158))
 * Enable authorization via Colab secrets ([#157](https://github.com/Kaggle/kagglehub/pull/157))
 
-## v.0.3.0 (September 4th, 2024)
+## v0.3.0 (September 4th, 2024)
 
 * Integrate Keras metrics ([#152](https://github.com/Kaggle/kagglehub/pull/152))
 * Enhance logging messages ([#151](https://github.com/Kaggle/kagglehub/pull/151))
 * Upgrade Python version support to 3.9 and above ([#154](https://github.com/Kaggle/kagglehub/pull/154))
 
-## v.0.2.9 (July 31st, 2024)
+## v0.2.9 (July 31st, 2024)
 
 * Improve upload by ignoring patterns ([#147](https://github.com/Kaggle/kagglehub/pull/147))
 * Upload with empty files ([#145](https://github.com/Kaggle/kagglehub/pull/145))
 * Attach dataset in Kaggle notebook environment ([#134](https://github.com/Kaggle/kagglehub/pull/134))
 * Upload datasets using `kagglehub.dataset_upload(...)` ([#149](https://github.com/Kaggle/kagglehub/pull/149))
 * Download datasets using `kagglehub.dataset_download(...)` ([#131](https://github.com/Kaggle/kagglehub/pull/131))
 
-## v.0.2.8 (July 16th, 2024)
+## v0.2.8 (July 16th, 2024)
 
 * Added support for implicit token auth in kaggle notebooks ([#141](https://github.com/Kaggle/kagglehub/pull/141))
 * Improve logging for cached models in Kaggle & Colab environments ([#142](https://github.com/Kaggle/kagglehub/pull/142))
 
-## v.0.2.7 (July 1st, 2024)
+## v0.2.7 (July 1st, 2024)
 * Fix model downloading bug
 
-## v.0.2.6 (June 10th, 2024)
+## v0.2.6 (June 10th, 2024)
```

Comment:
Removed the leading `.` in the version. Follows the existing pattern prior to this version.